flask_api.py

```diff
@@ -77,6 +79,7 @@ def handle_question():
     if "question" not in request_body:
         return "request body should include the question", BAD_REQUEST
 
+    nimbus = Nimbus()
```

Comment:
Another place to review for @cameron-toy

Reply:
I think this will break handle_question(), since the root url may not have been visited yet and so the object may not have been initialized.  I think this is used here: https://github.com/calpoly-csai/api/blob/dev/flask_api.py#L80

Reply:
Maybe put this in the constructor of the NimbusMySQLAlchemy class? So:

1. Create NimbusMySQLAlchemy object
2. Constructor of object creates a Nimbus object, passing itself in as the database (`Nimbus(this)`)
3. Constructor of Nimbus object passes the database to each QA object as they're constructed.

Reply:
Not quite clear to me how moving the initializing of the Nimbus class to the NimbusMySQLAlchemy class will help us here without additional changes - can you elaborate on how you plan to call the answer_question() function after abstracting away the creation of Nimbus?